python/fbprophet/tests/test_prophet.py

```diff
@@ -28,14 +28,59 @@
 
 class TestProphet(TestCase):
 
+    @staticmethod
+    def rmse(predictions, targets):
+        return np.sqrt(np.mean((predictions - targets) ** 2))
+
     def test_fit_predict(self):
+        days = 30
         N = DATA.shape[0]
-        train = DATA.head(N // 2)
-        future = DATA.tail(N // 2)
+        train = DATA.head(N - days)
```

Comment:
Is this mainly to make the tests more reliable?

Reply:
Yes, indeed.